### DISCUSSION TITLE
Landing Page Discussion

### DISCUSSION THREAD

**Discussion by ZzRanger** [acmutd/comet_planning#116] [Ideas]
Per #109, this will be a thread to discuss the landing page. Below are the things that need to get done:
1. Product designer needs to outline the functional requirements of the landing page
2. Designers needs to finalize the mock-up of the landing page

https://www.figma.com/file/cycV0bxP6vTJJYUONFoBr0/Planner-Mockups?node-id=24851%3A30432 contains the current mockups of the landing page. 

Also, if anyone has feedback for the landing page, please comment it down below.

**Comment by syl15** [comment]
### Thoughts/Feedback 
![image](https://user-images.githubusercontent.com/87793234/157708948-91daae8e-0e03-4825-8b2e-5707c0ea9fee.png)
I personally really like the current Landing page mockup but I've left a few questions/thoughts below 

_____


![image](https://user-images.githubusercontent.com/87793234/157709277-39c07426-59fb-40fb-8635-cc6e12b6bf2b.png)
+ I like having the planner logo but I think it would be good to have the Planner logo on the left and then Nebula right next to it on the right (similar to the login/sign up page). I think it would be better for branding.

______ 

![image](https://user-images.githubusercontent.com/87793234/157709355-6755b5d9-26cd-43ed-89bb-990305a66401.png)
+ I personally think that this part should have two options: "log in" or "continue as a guest" instead, with log in on the right. 
+ Once you get to log in, then have an option to sign up (I feel like this makes more intuitive sense than displaying sign up and continuing as a guest every time)
+ If we are differentiating between guest view and user authenticated view, what are the main differences in usage? I'd assume that students want to save plans so do we even want a guest option?

_______

![image](https://user-images.githubusercontent.com/87793234/157710299-780ca030-abb7-44b8-afcb-91f98d99a392.png)
+ For accessibility purposes, I think having a darker shadow around the "Planning made easy" would be a good idea. 
+ Is the right side scrollable or just a static object for display? (I'm leaning towards the latter being a better idea and maybe just adding some cool animation/transition if that's possible)

_______

+ for the navigation bar, since I suggested moving the login button to the left sidebar, we could maybe replace it with a help/feedback button, especially for the earlier releases where we need immediate feedback

**Comment by ZzRanger** [comment]
The team met yesterday to finalize designs for the landing page, which can be found below:

https://www.figma.com/file/cycV0bxP6vTJJYUONFoBr0/Planner-Mockups?node-id=24929%3A31445
